### DISCUSSION TITLE
External MIDI problem

### DISCUSSION THREAD

**Discussion by ll-dasein-ll** [KBLiveSolutions/open.control#28] [Troubleshooting]
Hi Pierre - [](url)

Hope you're well!  It's been a while since I've needed any help, but I'm back (sorry 😂).  Loving the OpenControl, I couldn't live without it any more, but I've got one problem that I can't get my head around - using the External MIDI function to control it with another device.

Apologies in advance for what's bound to be a long post, but I've just bought one of these: [https://www.tindie.com/products/distropolis/phantasmal-force-micro-midi-controller/](https://www.tindie.com/products/distropolis/phantasmal-force-micro-midi-controller/).  It has a TRS-A port at the back of it.  I've connected it to the OpenControl, have set up a button on the Force device to send CC#29 at velocity 127 on channel 9, and configured button 5 on the open control to be triggered by that CC, but I can't get it to work no matter what I do!  At the moment I'm just trying to trigger Play/Stop for testing purposes.

I've tried 3 different 3.5mm stereo audio TRS cables, and have ordered one of these [Boss TRS-A MIDI cables](https://www.pmtonline.co.uk/boss-bcc-1-3535-30cm-trs-to-trs-cable) in case the cables are the problem.  Using a MIDI monitor app I can see that button 5 on the Open Control is sending CC#1 on channel 16 to trigger Play/Stop in Ableton.  If I configure a button on the Force device to send that same signal (CC#1 on channel 16), then the LED on button 5 on the OpenControl lights up when the button is pressed, and if I change the velocity value of the CC signal then the colour of button 5's LED changes.  This suggests the signal is getting sent to the OpenControl successfully but it just won't trigger the actual button command no matter what settings I configure.

I was going to wait until the new cable arrives before posting this, but since that's going to be next week I'm wondering if you could suggest anything that I might not have tried yet?  So far I've tried different cables, have tried reversing the polarity of the signal via a couple of TRS to phono adaptors to flip it to TRS-B (in case the OpenControl uses TRS-B?), I've tried sending both CC and Note signals on different channels from the Force, tried rolling the firmware back to 1.4 on the OpenControl, tried deleting and re-adding OpenControl in my Mac's Audio MIDI Setup app, and also connecting the output of the Force device directly to the input of the OpenControl via the Audio MIDI Setup app.

If I connect the devices' inputs and outputs in Audio MIDI Setup, by using a MIDI Monitor app I can see that I can get the Mac to think it's receiving a signal from the OpenControl when I press a button on the Force and vice versa, and as I mentioned above, I can get the Force to trigger button 5's LED, just not the actual Play/Stop command.

Any help would be much appreciated as always, although I'm aware there's a chance this problem is due to the cables and the Boss one might be all I need.  I'd never come across TRS MIDI until I bought the OpenControl and I'm still not 100% sure if I'm right in thinking that standard stereo audio cables can be used for MIDI signals...

**Comment by KBLiveSolutions** [comment]
Hi there!
Glad to hear you're still enjoying your opencontrol!
So, audio cables won't work for MIDI, at least when going TRS to DIN. You have to get dedicated MIDI cables such as [these](https://www.thomann.de/fr/befaco_trs_midi_cable_a.htm) (this one actually works with opencontrol).
I have never tried TRS to TRS for MIDI, but since they sell such a cable, I guess it is required.
Now if you manage to connect your device and have the corresponding LED go white on opencontrol, I guess it means that opencontrol is well receiving the control. It should send the corresponding action.
I'll investigate on this when I have a moment.
Cheers
p

**Reply by ll-dasein-ll** [reply]
Thanks Pierre - I had a feeling the cables may have been the problem.  Thankfully I got a notification overnight saying the new one I've ordered should arrive today, so I'll not have a long a wait as I expected!  I'lll let you know if it works - fingers crossed! 🤞

**Reply by ll-dasein-ll** [reply]
Unfortunately the new MIDI TRS cable also doesn't work.  I really can't understand why!  To be honest it doesn't matter too much, I can manage without this feature (although it would be nice to figure out why it isn't working at some point).

I've also noticed something that may or may not be related: setting an LED to blink as a metronome no longer works for me, but it definitely used to the last time I tried it.  Could I have somehow accidentally corrupted something?  I've reinstalled the firmware a few times, going between v1.4 and 1.5 mainly, but the metronome LED function doesn't seem work with either of them any more.

**Reply by 77166** [reply]
Hi , It seems that I have the same type of problem with my pedalboard FCB 1010
  and the same, I'm waiting for another  Cable,  the problem may come from there.?…Does it work for you now …?

**Comment by ll-dasein-ll** [comment]
Unfortunately not - I never got this to work.  Signal does pass through - the LEDs respond to button presses (as I mentioned in my orignal post) but the actual intended actions do not work.

**Comment by 77166** [comment]
Hi,
Well, personally and finally thanks to Pierre Antoine who reviewed it, I can control Open Control with my FCB 1010 foot pedal. Both work well with the new firmware. In the editor and the external midi section, I favored notes rather than CCs. But, unfortunately on the other hand I followed the procedure which explains Pierre Antoine in his video on loopers and I still have a problem with the LED supposed to reflect the state of the last Looper. It is never synchronized and it does not display the correct status. This is not a midi conflict problem, because the problem persists even when open control is the only external device ????
